### PR TITLE
fix: remove dup item

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/asset-inventory/stores/cloud-service-detail-page-store.ts [hunk 1]
```diff
@@ -44,7 +44,13 @@ export const useCloudServiceDetailPageStore = defineStore('cloud-service-detail-
...
                     query: _getCloudServiceTypeQuery(this.provider, this.group),
                 });
-                this.cloudServiceTypeList = results ?? [];
+                this.cloudServiceTypeList = (results || []).reduce((acc: CloudServiceTypeModel[], cur) => {
+                    const idx = acc.findIndex((item) => item.cloud_service_type_key === cur.cloud_service_type_key);
+                    if (idx === -1) {
+                        acc.push(cur);
+                    }
+                    return acc;
+                }, []);
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/stores/cloud-service-detail-page-store.ts:53]: suggestion: how about to use `uniqBy()` from lodash?

```typescript
uniqBy(cloudServiceTypeList, 'cloud_service_type_key')
```

**Review by yuda110 — APPROVED** [review]
I left a comment!!!

File: apps/web/src/services/asset-inventory/stores/cloud-service-detail-page-store.ts (hunk 1 above)
Comment (piggggggggy) [reply apps/web/src/services/asset-inventory/stores/cloud-service-detail-page-store.ts:53]: Queen of lodash

**Review by piggggggggy — APPROVED** [review]
LGTM!

File: apps/web/src/services/asset-inventory/stores/cloud-service-detail-page-store.ts (hunk 1 above)
Comment (skdud4659) [reply apps/web/src/services/asset-inventory/stores/cloud-service-detail-page-store.ts:53]: Oh,,, good,,,,

**Pull request merged**